contracts/carbon/CarbonController.sol

```diff
@@ -26,9 +26,6 @@ contract CarbonController is
     OnlyProxyDelegate,
     Utils
 {
-    // the emergency manager role is required to pause/unpause
```

Comment:
`PausableUpgradeable` is no longer needed, but removing it will indeed affect the storage layout. One way to resolve it is to add a `bool private reserved0` (+ gap) on top of the storage (either explicitly or by replacing `PausableUpgradeable` with `ReservedXXX` that only reserved `50` slots and that's it

Reply:
thought it's a bit risky, but it's definitely doable.

Reply:
It is riskier than keeping it as is, but:
1. It's easy to verify that the layout was preserved (do we have the layout export command here as well?)
2. It's cleans the code a bit and doesn't keep unused/unneeded methods (but it's not the end of the world)

Reply:
removed